my_view_inc.php

```diff
@@ -351,7 +350,9 @@
 			print_icon( 'fa-square', 'fa-status-box ' . $t_status_css, $t_status );
 			echo ' ';
 
-			if( !bug_is_readonly( $t_bug->id ) && access_has_bug_level( $t_update_bug_threshold, $t_bug->id ) ) {
+			$t_can_update = !bug_is_readonly( $t_bug->id ) &&
+				access_has_bug_level( config_get( 'update_bug_threshold', null, $t_current_user_id, $t_bug->project_id  ), $t_bug->id );
```

Comment:
To make code more readable, I would keep the *$t_update_bug_threshold* variable
```php
$t_update_bug_threshold = config_get( 'update_bug_threshold', null, $t_current_user_id, $t_bug->project_id  );
$t_can_update = !bug_is_readonly( $t_bug->id ) && access_has_bug_level( $t_update_bug_threshold, $t_bug->id );
```

Reply:
I did it this way because of better performance / less server stressing (fewer calls of config_get in some cases)
@dregad what do you think, readability or performance?

Reply:
Ah yes, I see what you mean. Let's leave it like that then.